### ISSUE TITLE
librime编译错误

### ISSUE THREAD

**Issue by GoogleCodeExporter** [lotem/rimeime#271]
```
Linking CXX shared library ../lib/librime.so
/usr/bin/ld: ../lib/librime.a(key_event.cc.o): relocation R_X86_64_32S against 
`_ZNSs4_Rep20_S_empty_rep_storageE' can not be used when making a shared 
object; recompile with -fPIC
../lib/librime.a: could not read symbols: Bad value
collect2: error：ld return 1



1. 輸入法的款式：（請選擇）中州韻 / 鼠鬚管 / 小狼毫
輸入法版本號：（請填寫）

2. 操作系統類型：（請選擇）Linux 
   操作系統版本：（請填寫）archlinux


還想再說幾句：



```

Original issue reported on code.google.com by `fra...@gmail.com` on 26 Sep 2012 at 12:50

**Comment by GoogleCodeExporter** [comment]
Original comment by `chen....@gmail.com` on 26 Sep 2012 at 4:21
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
possibly fixed in 0.9.4. have a try.
```

Original comment by `chen....@gmail.com` on 26 Sep 2012 at 5:24
- Changed state: **Fixed**
